js/ui/grid_core/ui.grid_core.rows.js

```diff
@@ -664,7 +682,7 @@ module.exports = {
                         .toggleClass(COLUMN_LINES_CLASS, that.option("showColumnLines"));
 
                     for(i = 0; i < columns.length; i++) {
-                        freeSpaceRowElement.append(that._createCell({ column: columns[i], rowType: "freeSpace", columnIndex: i, columns: columns }));
+                        freeSpaceRowElement.append(that._createCell({ column: columns[i], rowType: "freeSpace" }));
```

Comment:
Why? This change may course problems with the new advancedRendering.

Reply:
My fault, will roll back it